public/prod/lunchroom_manners.json

```
@@ -257,7 +257,7 @@
                   ]
                 }
               ],
-              "target": "/prod/lunchroom_manners/canvas/1"
+              "target": "https://iiif-react-media-player.netlify.app//prod/lunchroom_manners/canvas/1"
```

Looks like there's an extra slash here and at ln 284

```suggestion
              "target": "https://iiif-react-media-player.netlify.app/prod/lunchroom_manners/canvas/1"
```

---

Good catch :100: